package.json

```diff
@@ -38,9 +38,10 @@
     "@babel/core": "^7.23.0",
     "@babel/eslint-parser": "^7.25.9",
     "@babel/preset-env": "^7.22.20",
-    "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#1.3.0",
+    "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#8.1.0",
     "@duckduckgo/eslint-config": "github:duckduckgo/eslint-config#v0.1.0",
     "@duckduckgo/privacy-test-pages": "github:duckduckgo/privacy-test-pages#1.3.1",
+    "@duckduckgo/privacy-configuration": "github:duckduckgo/privacy-configuration#dbajpeyi/add-new-autofill-config",
```

Comment:
I guess this will need to be changed before merging. Let's make sure we don't forget.